0017-chart-namespace-overrides/README.md

```diff
@@ -0,0 +1,440 @@
+
+#### Reason for Rejection
+
+This is one of the more "Zarf-way" options however, this would be the only place in the Zarf package definition that we allowed the `###ZARF_VAR` syntax to exist and the ### syntax is not very conducive to being maintained in a Zarf package (i.e. it is  treated as a comment when unquoted adding complexity for users and/or when trying to parse Zarf packages since this cannot be pre-templated like other `###ZARF_PKG_TMPL`s).
+
+### Option 4 (Prefix/Suffix Style)
+
+**Given** I have a Zarf Package with a chart deploying to `my-namespace`
+**When** I deploy that package with a `zarf-config.yaml` like the below*:
+```yaml
+package:
+  deploy:
+    namespace-prefix: new-
+    namespace-suffix: -kitteh
+```
+**Then** Zarf will change the chart's release namespace to `new-my-namespace-kitteh`
+
+**Given** I have a Zarf Package with a chart deploying to `my-namespace`
+**When** I deploy that package with a `--set` like the below:
+```bash
+zarf package deploy zarf-package-test.tar.zst --namespace-prefix new- --namespace-suffix -kitteh
+```
+**Then** Zarf will change the chart's release namespace to `new-my-namespace-kitteh`
+
+#### Reason for Rejection
+
+This would allow for some customization but may not provide enough flexibility in some cases.  Some clusters for security reasons will only authorize the deploy user to access specific namespaces which may not line up with what was originally in the package.  It also does not allow namespaces to be easily present elsewhere such as in `actions`.
+
+### Option 5 (Package Namespace Style)
+
+**Given** I have a Zarf Package that implements a new package `namespace` field
+```
+kind: ZarfPackageConfig
+metadata:
+  name: test
+  namespace: my-namespace
+```
+**And** That package has `chart` and `manifest` resources that omit `namespace`
+```
+ charts:
+  - name: my-chart
+    url: oci://...
+```
+**When** I deploy that package with a `zarf-config.yaml` like the below*:
+```yaml
+package:
+  deploy:
+    namespace: new-namespace
+```
+**Then** Zarf will change the chart's release namespace to `new-namespace`
+
+**Given** I have a Zarf Package that implements a new package `namespace` field
+```
+kind: ZarfPackageConfig
+metadata:
+  name: my-package
+  namespace: my-namespace
+```
+**And** That package has `chart` and `manifest` resources that omit `namespace`
+```
+ charts:
+  - name: my-chart
+    url: oci://...
+```
+**When** I deploy that package with `--namespace` like the below:
+```bash
+zarf package deploy zarf-package-test.tar.zst --namespace new-namespace
+```
+**Then** Zarf will change the chart's release namespace to `new-namespace`
+
+#### Reason for Rejection
+
+While this is the most "Helm-way" option, many packages have multiple namespaces they need to deploy to (even simpler ones like `uds-package-postgres-operator`).  This would then at least require some templating to make work, but again that could have the same drawbacks as prefixes / suffixes.
```

Comment:
After reading this proposal again this feels like it may be the ideal way. In the current proposal if a namespace is unedited wouldn't a package open itself up to 

It feels like the current proposal doesn't handle multiple namespaces well anyway, so this feels like the more intuitive option.